library/tools/styles/script.js

```diff
@@ -0,0 +1,77 @@
+const postcss = require('postcss');
```

Comment:
Can we come up with a better name for this file? `script` seems too broad and doesn't say anything. Maybe call it `theme-generator` or `css-generator` or `styles-generator` or ...

Reply:
You're right, I changed to `theme-generator`, thanks!